File: app/src/main/java/dina/nota/app/model/data/Note.java
package dina.nota.app.model.data;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "note_table")
public class Note {

    @PrimaryKey(autoGenerate = true)
    private int id;

    private String title, description, soundFilePath;

    public Note(String title, String description, String soundFilePath) {
        this.title = title;
        this.description = description;
        this.soundFilePath = soundFilePath;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }


    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public String getSoundFilePath() {
        return soundFilePath;
    }
}

File: app/src/main/java/dina/nota/app/data/room/Note.kt
package dina.nota.app.data.room

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "notes_table")
class Note {
    private val noteTitle: String
    @PrimaryKey
    private val id: Int
    private val noteBody: String

    constructor(noteTitle: String, id: Int, noteBody: String) {
        this.noteTitle = noteTitle
        this.id = id
        this.noteBody = noteBody
    }
}